scripts/test-py3.sh

```diff
@@ -5,14 +5,12 @@ pytest openlibrary/catalog openlibrary/coverstore openlibrary/mocks openlibrary/
        --ignore=openlibrary/catalog/marc/tests/test_get_subjects.py \
        --ignore=openlibrary/catalog/marc/tests/test_parse.py \
        --ignore=openlibrary/tests/catalog/test_get_ia.py \
-       --ignore=openlibrary/coverstore/tests/test_doctests.py \
        --ignore=openlibrary/plugins/openlibrary/tests/test_home.py
 RETURN_CODE=$?
        
 pytest openlibrary/catalog/marc/tests/test_get_subjects.py || true
 pytest openlibrary/catalog/marc/tests/test_parse.py || true
 pytest openlibrary/tests/catalog/test_get_ia.py || true
-pytest --show-capture=all openlibrary/coverstore/tests/test_doctests.py || true
```

Comment:
Same as above, yes -- this _enables_ the test battery? If so, excellent.

I wonder if it'd be nice to reorganize this list of files into a variable, so that it's clearer in this file, but it's not necessary.

Reply:
I vote that we get all the tests to pass instead of tidying up this temporary script.